_books/ion-1-1/src/modules/system_module.md

```diff
@@ -20,72 +20,70 @@ The Ion 1.1 System Symbol table _replaces_ rather than extends the Ion 1.0 Syste
 <!-- make the tables align to the side of the page /-->
 <style>table { margin: 1em;}</style>
 
-| ID | Text                                           |
-|---:|:-----------------------------------------------|
-|  1 | `$ion`                                         |
-|  2 | `$ion_1_0`                                     |
-|  3 | `$ion_symbol_table`                            |
-|  4 | `name`                                         |
-|  5 | `version`                                      |
-|  6 | `imports`                                      |
-|  7 | `symbols`                                      |
-|  8 | `max_id`                                       |
-|  9 | `$ion_shared_symbol_table`                     |
-| 10 | `$ion_encoding`                                |
-| 11 | `$ion_literal`                                 |
-| 12 | `$ion_shared_module`                           |
-| 13 | `macro`                                        |
-| 14 | `macro_table`                                  |
-| 15 | `symbol_table`                                 |
-| 16 | `module`                                       |
-| 17 | `retain`                                       |
-| 18 | `export`                                       |
-| 19 | `catalog_key`                                  |
-| 20 | `use`                                          |
-| 21 | `load`                                         |
-| 22 | `import`                                       |
-| 23 | _&lt;empty string>_ (i.e. `''`)                |
-| 24 | `literal`                                      |
-| 25 | `if_void`                                      |
-| 26 | `if_single`                                    |
-| 27 | `if_multi`                                     |
-| 28 | `for`                                          |
-| 29 | `fail`                                         |
-| 30 | `values`                                       |
-| 31 | `annotate`                                     |
-| 32 | `make_string`                                  |
-| 33 | `make_symbol`                                  |
-| 34 | `make_blob`                                    |
-| 35 | `make_decimal`                                 |
-| 36 | `make_timestamp`                               |
-| 37 | `make_list`                                    |
-| 38 | `make_sexp`                                    |
-| 39 | `make_struct`                                  |
-| 40 | `parse_ion`                                    |
-| 41 | `repeat`                                       |
-| 42 | `delta`                                        |
-| 43 | `flatten`                                      |
-| 44 | `sum`                                          |
-| 45 | `local_symtab` (or maybe just `symbol_table`?) |
-| 46 | `lst_append` (or maybe just `add_symbols`?)    |
-| 47 | `local_mactab` (or maybe just `macro_table`?)  |
-| 48 | `lmt_append` (or maybe just `add_macro`?)      |
```

Comment:
🗺️ I used `symbol_table`, `macro_table`, `add_symbols`, and `add_macros`. The re-flow caused the entire table to appear in the diff.